README.md

```diff
@@ -109,48 +146,55 @@ python -m tf2onnx.convert
     [--verbose]
 ```
 
-## Parameters
-### --input or --graphdef
-TensorFlow model as graphdef file. If not already frozen we'll try to freeze the model.
-More information about freezing can be found here: [freeze graph tool](#freeze_graph).
-### --checkpoint
-TensorFlow model as checkpoint. We expect the path to the .meta file. tf2onnx will try to freeze the graph.
-### --saved-model
-TensorFlow model as saved_model. We expect the path to the saved_model directory. tf2onnx will try to freeze the graph.
-### --output 
-the target onnx file path.
-### --inputs, --outputs
-Tensorflow model's input/output names, which can be found with [summarize graph tool](#summarize_graph). Those names typically end on ```:0```, for example ```--inputs input0:0,input1:0```. inputs and outputs are ***not*** needed for models in saved-model format.
-### --inputs-as-nchw
-By default we preserve the image format of inputs (nchw or nhwc) as given in the TensorFlow model. If your hosts (for example windows) native format nchw and the model is written for nhwc, ```--inputs-as-nchw``` tensorflow-onnx will transpose the input. Doing so is convinient for the application and the converter in many cases can optimize the transpose away. For example ```--inputs input0:0,input1:0 --inputs-as-nchw input0:0``` assumes that images are passed into ```input0:0``` as nchw while the TensorFlow model given uses nhwc.
-### --opset
+### Parameters
+
+#### --saved-model
+
+TensorFlow model as saved_model. We expect the path to the saved_model directory.
+
+#### --input or --graphdef
+
+TensorFlow model as graphdef file.
+
+#### --checkpoint
+
+TensorFlow model as checkpoint. We expect the path to the .meta file.
+
```

Comment:
Can we keep the ordering of the arguments the same as the command line above?  Makes it easier to connect the dots visually.

Reply:
Yes, sure, that makes sense. I'll also try to clarify that only one of the model format options should be provided

Reply:
Fixed